Services/IotHub/CheckHubPermissionsWrapper.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+
+using Microsoft.Azure.Devices;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.IotHub
+{
+    public interface ICheckHubPermissionsWrapper
```

Comment:
we use the "Wrapper" suffix when wrapping a class or that doesn't support testing, e.g. some of the SDK classes. I would change the naming here to follow the practice:
1. no need for this new class
2. add a new `GetJobsAsync` method in `RegistryManagerWrapper` - notice how that class is simply a proxy isolating the real code for testability, and the name of the wrapping method is about the same of the original method being called
3. in `IotHubConnectionStringManager.TestIoTHubReadPermissionsAsync()` call `GetJobsAsync` -- please add a comment explaining why, e.g. mentioning that we make a call not affecting the quota used by the simulation agent, to avoid interfering with simulations in progress

Reply:
1. removed the new class
2. added GetJobsAsync function to RegistryManagerWrapper
3. Added a comment explaining why